docker/dev/ubuntu/scripts/examples/mongo.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+docker --tlsverify=false run -d --name mongoOrion mongo:2.6.10 
```

Comment:
I woud suggest a docker compose file instead of this one. Or at least in addition to this one.